### ISSUE TITLE
Some issue exposed by Math::FFT::Libfftw3

### ISSUE THREAD

**Issue by AlexDaniel** [rakudo/rakudo#2682]
See https://github.com/perl6/ecosystem-unbitrot/issues/468

Math::FFT::Libfftw3 – Fail, Bisected: [rakudo/rakudo@ccf350e](https://github.com/rakudo/rakudo/commit/ccf350e4bcf9a70f4038cb4259f82fc913dd87cd)

Ping @lizmat.

**Comment by dogbert17** [comment]
Fixed by lizmat++ with 0d2658b806de11a538d69080737c59ea9067c8dc

**Comment by Altai-man** [comment]
So the issue was that `[ ]` postcircumfix for whateverstar was using an obsolete method call when called for CArray. The golf is:

```
    my @t2682 := CArray[int32].new(1,2,3);
    is-deeply @t2682[0..*-2], (1,2), 'Indexing with WhateverStar works on CArray';
```

^ this is added as a rakudo test.
